webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -12,6 +12,13 @@ const locales = window.locales || emptyLocales;
 let locale = document.getElementsByTagName('html')[0].lang;
 
 locale = locale.length === 0 ? 'en' : locale.replace(/-/g, '_');
+
+if (locales[locale] === undefined) {
+  // eslint-disable-next-line no-console
+  console.log(`could not load translations for ${locale} locale, falling back to English`);
+  locale = 'en';
+}
```

Comment:
last nitpick if you may, could you extract 'en' as a const e.g.
```js
const DEFAULT_LOCALE='en';

locale = locale.length === 0 ? DEFAULT_LOCALE : locale.replace(/-/g, '_');
￼
if (locales[locale] === undefined) {
￼   // eslint-disable-next-line no-console
￼   console.log(`could not load translations for ${locale} locale, falling back to English`);
￼   locale = DEFAULT_LOCALE;
￼ }
￼```

Reply:
changed